src/components/Form/Form.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCallback } from 'react';
+import './Form.scss';
+
+type Props = {
+  searchParams: any,
+  navigate: any,
+  sortQuery: string,
+  filterQuery: string,
+};
+
+export const Form: React.FC<Props> = ({
+  searchParams,
+  navigate,
+  sortQuery,
+  filterQuery,
+}) => {
+  const handleInputChange = useCallback((value: any) => {
+    if (value.length > 0) {
```

Comment:
```suggestion
    if (value.trim().length > 0) {
```
https://user-images.githubusercontent.com/38065883/150570798-3b315add-d5e2-4583-996c-b305aa706cd2.mov